packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -60,6 +64,13 @@ export interface CameraAndMicrophoneDomainPermissionsProps extends CommonDomainP
   strings?: CameraAndMicrophoneDomainPermissionsStrings;
 }
 
+/* @conditional-compile-remove(call-readiness) */
+const isSafari = (browser: string | undefined): boolean => {
+  /* @conditional-compile-remove(unsupported-browser) */
+  return browser === 'safari';
+  return /^((?!chrome|android|crios|fxios).)*safari/i.test(navigator.userAgent);
```

Comment:
This regex check is in two places, consider exporting (internally) a util function called `_isSafari`:
```
export const _isSafari = (environmentInfo: /** conditional compile environment info type out */ EnvironmentInfo | undefined) {
  /* @conditional-compile-remove(unsupported-browser) */
  return environmentInfo?.environment.browser === 'safari';
  return /^((?!chrome|android|crios|fxios).)*safari/i.test(navigator.userAgent);
}`

Reply:
Updated